.github/workflows/autotest.yml

```diff
@@ -5,7 +5,16 @@ on:
   - push
 
 jobs:
+  add_header_test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - run: |
+          tools/add_header --disable-add-me -cd -- bin/* $(find . -type f -name '*.py')
```

Comment:
the max length of a shell command is limited. This commands risks to fail with the growth of the project.

Reply:
Yeah, totally forgot about that. We currently are at about 0.5% of the maximum line length. Changed it never the less.